client/my-sites/checkout/checkout-thank-you/domain-transfer-to-any-user/index.tsx

```diff
@@ -34,18 +34,16 @@ const DomainTransferToAnyUser: React.FC< DomainTransferToAnyUserContainerProps >
 		),
 	};
 
-	const products = [ <ThankYouDomainProduct domainName={ domain } key="transferred-domain" /> ];
-
 	return (
-		<Main className="is-redesign-v2">
+		<Main className="checkout-thank-you is-redesign-v2">
 			<MasterbarStyled canGoBack={ false } />
 
 			<ThankYouV2
 				title={ translate( 'Your domain transfer is underway' ) }
 				subtitle={ translate(
 					'Domain transfers can take a few minutes, we’ll email you once it’s set up.'
 				) }
-				products={ products }
+				products={ <ThankYouDomainProduct domainName={ domain } key="transferred-domain" /> }
```

Comment:
```suggestion
				products={ <ThankYouDomainProduct domainName={ domain } /> }
```

`key` is necessary for arrays